jetty-http/src/main/java/org/eclipse/jetty/http/MultiPartFormInputStream.java

```diff
@@ -123,8 +122,8 @@ protected void close() throws IOException
         {
             _out.close();
         }
-        
-        protected void write(int b) throws IOException
+
+        protected void write(byte b) throws IOException
```

Comment:
yes, this is just part of the cleanups I was doing for the `MultiPartFormInputStream`, this method is not used but I think it is correct for it to be taking a byte and not an int

Reply:
@lachlan-roberts please revert this change, as all the stream APIs (`OutputStream`, etc.) to write a single byte take an `int`, not a `byte`.
Furthermore, this single-byte write must be implemented to forward the call to `write(byte[], int, int)` to avoid code duplication.